content/50.discussion.md

```diff
@@ -11,8 +11,42 @@ In this review, we seek to explain the scientific rationale underlying these tec
 
 ### Concerns about Equity in Healthcare
 
-Scientific and medical research broadly is shaped by a number of biases.
-Some concerns include how clinical trials recruit and operate.
+The objective of the paper is to compile and synthesis literature on the disease COVID-19.
+To accurately and completely discuss this pandemic, health disparities and bias need to be addressed, so that the information is applicable to all.
+Bias in medicine can be characterized as when a physician and/or patient “others” an individual in a healthcare setting, oftentimes this occurs when a person looks or acts differently.
+By being self-aware of our own implicit biases, one can work towards equality.
 
-<!--Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.-->
+The population of the inhabitants or humans on Earth is approximately 7.8 billion people.
+The myriad of cultures, languages, and environments differentiate the population into subpopulations with varying beliefs, values, and point of views.
+However, the preponderance of the academic discourse made public is produced by western, educated, industrialized, rich and democratic, or WIERD, individuals, which comprise 12 percent of world occupants.
+While there is not much empirical data on the representativeness of WEIRD individuals within mankind, this blatant disparity brings forth the notion of bias in science, for researchers generalize their discoveries to the entire human population.
```

Comment:
Let's try to figure out what the clinical trials terminology for this is -- I think WEIRD is mostly used in psych. This seems like it might be helpful: https://acsjournals.onlinelibrary.wiley.com/doi/full/10.1002/cncr.23157

Reply:
I have a hard time seeing how some of this applies to COVID-19 specifically.  Are you concerned that there aren't enough COVID-19 trials in some regions of the world?  If so, it would be much more informative *in this context* to support that claim with data or citations, instead of just making sweeping and unsupported statements about how science in general is biased.  I'm not saying that you are wrong in what you're saying, but I think that in a review written by many people it's necessary to work extra hard to keep from becoming a collection of forty brief essays by forty different people.  And most importantly, I think that it is very important that these claims need to have citations to back them up.